mscore/musescore.h

```diff
@@ -820,6 +820,7 @@ class MuseScore : public QMainWindow, public MuseScoreCore {
       void unregisterPlugin(PluginDescription*);
 
       Q_INVOKABLE void showStartcenter(bool);
+      void reDisplayDockWidget(QDockWidget* widget, bool visible);
```

Comment:
Since "redisplay" is one word, I would capitalize this function name as "redisplayDockWidget".

Reply:
I see your rationale. I chose to do it that way because I think it's more readable and makes the main verb (display) stand out more clearly.